configuration/items.md

```diff
@@ -271,6 +271,13 @@ Number    Livingroom_Clock_Battery "Battery Charge [%d %%]"            // e.g. "
 Location  My_Location              "My Location [%2$s°N %3$s°E %1$sm]" // e.g. "49.26°N 123.19°E 0m"
 ```
 
+##### State Description
+
+Internally items have a State Description, which can be obtained by GET `/rest/items/<itemName>`.
+When an item is linked to a channel without `CommandTopic`, the State Description of the item is read-only.
+Some UIs may disallow the user to edit such an item.
```

Comment:
This sentence is wrong. Item editing does not depend on the state description.

Reply:
In OH-Android it does depend - https://github.com/openhab/openhab-android/issues/3870.

Reply:
I think your wording is wrong and hence we are talking about different things.
Item editing is actually modifying the config. You mean sending commands, not editing.

Reply:
I changed it:
```diff
@@ -275,7 +275,7 @@ Location  My_Location              "My Location [%2$s°N %3$s°E %1$sm]" // e.g.
 
 Internally items have a State Description, which can be obtained by GET `/rest/items/<itemName>`.
 When an item is linked to a channel without `CommandTopic`, the State Description of the item is read-only.
-Some UIs may disallow the user to edit such an item.
+Some UIs may disallow the user to send commands to such an item.
 To change the item to read-write add `stateDescription=" "[readOnly=false]` in bindingconfig.
 
 #### State Transformation
```